components/scripts/gradle/05-validate-remote-build-caching-ci-local.sh

```diff
@@ -369,11 +384,30 @@ EOF
   print_interactive_text "${text}"
 }
 
+collect_remote_build_cache() {
+  collect_remote_build_cache_type
+  collect_remote_build_cache_url
+}
+
+collect_remote_build_cache_type() {
+  local default_remote_cache_type="<project default>"
+  prompt_for_setting "What is the remote build cache type to use? [develocity, gradle-enterprise, http, or <BLANK>]" "${remote_build_cache_type}" "${default_remote_cache_type}" remote_build_cache_type
```

Comment:
The Gradle documentation and the Develocity Gradle plugin end-user documentation calls these "Build cache connectors".  I wonder if we should replace "type" with "connector" so that the BVS are consistent with the docs.

Reply:
It's a good point. 'connector' would be the correct term. Also, build cache type sounds like the type of the build cache -- and not like the class type used to coonect to the remote build cache.